aten/src/ATen/Context.h

```diff
@@ -38,6 +38,8 @@ class TORCH_API Context {
       return at::detail::getDefaultCPUGenerator();
     } else if (device_type == at::kCUDA) {
       return at::detail::getCUDAHooks().getDefaultCUDAGenerator(device.index());
+    } else if (device_type == at::kMPS) {
+      return at::detail::getMPSHooks().getDefaultMPSGenerator();
```

Comment:
Let's pass the index anyway..
```suggestion
      return at::detail::getMPSHooks().getDefaultMPSGenerator(device.index());
```

Reply:
Please see my comment below, and let me know if you'd think it's better to add it now or later when we support multiple MPS devices.

Reply:
Adding device index down to MPSGenerator is a bigger change (it's not just `getDefaultMPSGenerator()`), and I'll do that in a separate PR.